tests/bmtestspolar/test.m

```diff
@@ -0,0 +1,42 @@
+function test()
+    n = 100;
+    r = 5;
+    
+    A = randn(n,n);
+    A = 0.5*(A + A');
+    
+    problem.M = symfixedrankpolarfactory(n, r);
+    
+    problem.cost = @cost;
+    problem.egrad = @egrad;
+    problem.ehess = @ehess;
+    
+    symm = @(a) 0.5*(a+a');
+    
+    function f = cost(x)
+        f = 0.5*norm(x.U*x.B*x.U' - A, 'fro')^2;
+    end
+    
+    function g = egrad(x)
+        Res = x.U*x.B*x.U' - A;
```

Comment:
This could be replaced with a call to M.pair2matrix(x) for example.

it'd be good to also include a comment here to let the users know that this is not the proper way to work on this manifold: one should ideally never create the full-sized matrices. This here is just for testing purposes.

Reply:
The test file was for internal review only in order to check correctness. It should not be a part of the examples folder.